lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -1,9 +1,35 @@
+<%namespace name='static' file='/static_content.html'/>
 <%page args="section_data" expression_filter="h"/>
 <%!
 from django.utils.translation import ugettext as _
 from openedx.core.djangolib.markup import HTML, Text
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 %>
 
+% if configuration_helpers.get_value('ENABLE_SENSITIVE_DATA_MSG_ON_DASHBOARD', settings.FEATURES.get('ENABLE_SENSITIVE_DATA_MSG_ON_DASHBOARD', True)):
+
+<%static:require_module_async module_name="js/vendor/jquery.leanModal" class_name="leanModal">
+
+$( document ).ready(function() {
+  $('[data-section="data_download"]').click(function(){
+      $( 'a[rel*=leanModal]' ).leanModal({ top : 200, closeButton: ".modal_close" });
+      $( "#sensitive_data" ).click();
+  });
+});
+
```

Comment:
Is not necessary write `function (require) {...` as we talk before, since the `<%static...` tag place it by itself, if you go to source code in the browser you'll see this:

```
<script type="text/javascript">
  (function (require) {
  (function (require) {
  require(['js/vendor/jquery.leanModal'], function (leanModal) { 
  $( document ).ready(function() {
  $('[data-section="data_download"]').click(function(){
  $( 'a[rel*=leanModal]' ).leanModal({ top : 200, closeButton: ".modal_close" });
  $( "#sensitive_data" ).click();
  });
  });  
  });
  }).call(this, require \|\| RequireJS.require);
  }).call(this, require \|\| RequireJS.require);
  </script>
```
But if you note, ` (function (require) {...`  appears twice, same happens with other function already configured:

```
<%static:require_module_async module_name="js/dateutil_factory" class_name="DateUtilFactory">
    DateUtilFactory.transform(iterationKey=".localized-datetime");
</%static:require_module_async>
```
Which render this:

```
<script type="text/javascript">
  (function (require) {
  (function (require) {
  require(['js/dateutil_factory'], function (DateUtilFactory) {  
  DateUtilFactory.transform(iterationKey=".localized-datetime");  
  });
  }).call(this, require \|\| RequireJS.require);
  }).call(this, require \|\| RequireJS.require);
  </script>
```



It is normal or I'm missing something?